[BE]Clarify how to check memory saving if using gradient_as_bucket_view

zhaojuanmao opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #71483
* #71459

claify that peak memory saving should be checked after first iteration when using gradient_as_bucket_view

Differential Revision: [D33662424](https://our.internmc.facebook.com/intern/diff/D33662424/)

zhaojuanmao committed:
[BE]Clarify how to check memory saving if using gradient_as_bucket_view

claify that peak memory saving should be checked after first iteration when using gradient_as_bucket_view

Differential Revision: [D33662424](https://our.internmc.facebook.com/intern/diff/D33662424/)

[ghstack-poisoned]

rohan-varma reviewed (approved):
Thank you! Eventually, I think we should have a tutorial/docs around how users can check memory consumption of DDP, FSDP and what are the recommended memory savings techniques!

zhaojuanmao committed:
Update on "[BE]Clarify how to check memory saving if using gradient_as_bucket_view"


claify that peak memory saving should be checked after first iteration when using gradient_as_bucket_view

Differential Revision: [D33662424](https://our.internmc.facebook.com/intern/diff/D33662424/)

[ghstack-poisoned]
